samples/convert-upload-images-powervs/convert-upload-images-powervs

```diff
@@ -236,6 +237,17 @@ function apply {
   vm_access_info && success "Congratulations! create command completed"
 }
 
+#-------------------------------------------------------------------------
+# Read the info from the plan file
+#-------------------------------------------------------------------------
+function plan_info {
```

Comment:
where this function being used ?

Reply:
The function `plan_info` is already defined in `openshift-install-powervs` [L452-L458](https://github.com/ocp-power-automation/openshift-install-power/blob/devel/openshift-install-powervs#L452-L458) script, as we're only creating bastion vm, grep commands used for other variables are exiting the script. To resolve this issue I have overridden the script of this function `wc -l` command .